app/models/classification.rb

```diff
@@ -445,7 +442,7 @@ def self.seed
       category = find_by_name(c[:name], my_region_number, (c[:ns] || DEFAULT_NAMESPACE))
       next if category
 
-      category = new(c.except(:entries))
+      category = is_category.new(c.except(:entries))
```

Comment:
This will correctly set parent_id to 0/nil, so perhaps you want to also drop `where(:parent_id => nil).update_all(:parent_id => 0)` fixup done at the end?

Reply:
@cben thanks. That is a little scary, but I will test this out